austation/code/modules/pool/pool_main.dm

```diff
@@ -149,6 +149,19 @@
 										"<span class='userdanger'>You fall in the drained pool, but you had an helmet!</span>")
 					H.Knockdown(40)
 					playsound(src, 'sound/effects/woodhit.ogg', 60, TRUE, 1)
+			if(issilicon(AM))
+				if(!filled)
+					var/mob/living/silicon/robot/R = victim
+					if(prob(75))
+						R.Paralyze(75)
+						R.apply_damage(7, BRUTE)
+						R.visible_message("<span class='danger'>[R] falls in the drained pool!</span>",
+													"<span class='userdanger'>You fall in the drained pool!</span>")
+					else
+						R.Paralyze(200) //bad silicon, no powercreep
+						R.apply_damage(15, BRUTE)
+						R.visible_message("<span class='danger'>[R] falls in the drained pool and hits their module hard!</span>",
```

Comment:
"hits their module"
Wha

Reply:
7.5-20 seconds of _**paralyze**_
fucking Jesus that's annoying
at least carbons get to crawl out

Reply:
also, this lets carbons perma-stun and kill borgs since they can just mouse-click drag them off and on the pool surface, dealing damage and infinite stun.

Reply:
Carbons also have a chance of 20 sec stuns

Reply:
This is a punishment for borgs who even drained the pool in the first place

Reply:
Since only silicons can drain pool

Reply:
Unless it’s emagged, which is the same as killing with the holodeck

Reply:
actually, you can drain the pool by cutting a wire.